controllers/RestaurantController.ts

```diff
@@ -11,7 +11,7 @@ import RestaurantControllerI from "../interfaces/RestaurantControllerI";
  * <ul>
  *     <li>GET /api/restaurants to retrieve all the restaurant instances</li>
  *     <li>GET /api/restaurants/:rid to retrieve a particular restaurant instance</li>
- *     <li>POST /api/restaurants to create a new restaurant instance</li>
+ *     <li>POST /api/users/:uid/restaurants to create a new restaurant instance</li>
```

Comment:
Also mentioned in Yutong's pull request - path for creating a restaurant doesn't include users/:uid

Reply:
I added default picture in my branch